
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Computational Biology & Bioinformatics in the Zürich Area">
      
      
        <meta name="author" content="Michael Baudis">
      
      
      
      
      
      <link rel="icon" href="../img/favicon.ico">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.4.3">
    
    
      
        <title>News, Events, Seminars ... - CompbioZurich</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.79e020e9.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.a5377069.min.css">
      
      
  
  
    
    
  
  
  <style>:root{--md-admonition-icon--example:url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3.604 3.089A.75.75 0 0 1 4 3.75V8.5h.75a.75.75 0 0 1 0 1.5h-3a.75.75 0 1 1 0-1.5h.75V5.151l-.334.223a.75.75 0 0 1-.832-1.248l1.5-1a.75.75 0 0 1 .77-.037ZM8.75 5.5a.75.75 0 0 0 0 1.5h11.5a.75.75 0 0 0 0-1.5H8.75Zm0 6a.75.75 0 0 0 0 1.5h11.5a.75.75 0 0 0 0-1.5H8.75Zm0 6a.75.75 0 0 0 0 1.5h11.5a.75.75 0 0 0 0-1.5H8.75ZM5.5 15.75c0-.704-.271-1.286-.72-1.686a2.302 2.302 0 0 0-1.53-.564c-.535 0-1.094.178-1.53.565-.449.399-.72.982-.72 1.685a.75.75 0 0 0 1.5 0c0-.296.104-.464.217-.564A.805.805 0 0 1 3.25 15c.215 0 .406.072.533.185.113.101.217.268.217.565 0 .332-.069.48-.21.657-.092.113-.216.24-.403.419l-.147.14c-.152.144-.33.313-.52.504l-1.5 1.5a.75.75 0 0 0-.22.53v.25c0 .414.336.75.75.75H5A.75.75 0 0 0 5 19H3.31l.47-.47c.176-.176.333-.324.48-.465l.165-.156a5.98 5.98 0 0 0 .536-.566c.358-.447.539-.925.539-1.593Z"/></svg>');}</style>



    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../css/theme_overrides.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="purple" data-md-color-accent="indigo">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#news-events-seminars" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="CompbioZurich" class="md-header__button md-logo" aria-label="CompbioZurich" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CompbioZurich
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              News, Events, Seminars ...
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
          <a href="javascript:void(0)" class="md-search__icon md-icon" title="Share" aria-label="Share" data-clipboard data-clipboard-text="" data-md-component="search-share" tabindex="-1">
            
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M18 16.08c-.76 0-1.44.3-1.96.77L8.91 12.7c.05-.23.09-.46.09-.7 0-.24-.04-.47-.09-.7l7.05-4.11c.54.5 1.25.81 2.04.81a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3c0 .24.04.47.09.7L8.04 9.81C7.5 9.31 6.79 9 6 9a3 3 0 0 0-3 3 3 3 0 0 0 3 3c.79 0 1.5-.31 2.04-.81l7.12 4.15c-.05.21-.08.43-.08.66 0 1.61 1.31 2.91 2.92 2.91 1.61 0 2.92-1.3 2.92-2.91A2.92 2.92 0 0 0 18 16.08Z"/></svg>
          </a>
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/compbiozurich/compbiozurich.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 480 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M186.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1zM480 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2zm-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3zm-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1z"/></svg>
  </div>
  <div class="md-source__repository">
    compbiozurich.github.io
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="CompbioZurich" class="md-nav__button md-logo" aria-label="CompbioZurich" data-md-component="logo">
      
  <img src="../img/site_logo.png" alt="logo">

    </a>
    CompbioZurich
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/compbiozurich/compbiozurich.github.io" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 480 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M186.1 328.7c0 20.9-10.9 55.1-36.7 55.1s-36.7-34.2-36.7-55.1 10.9-55.1 36.7-55.1 36.7 34.2 36.7 55.1zM480 278.2c0 31.9-3.2 65.7-17.5 95-37.9 76.6-142.1 74.8-216.7 74.8-75.8 0-186.2 2.7-225.6-74.8-14.6-29-20.2-63.1-20.2-95 0-41.9 13.9-81.5 41.5-113.6-5.2-15.8-7.7-32.4-7.7-48.8 0-21.5 4.9-32.3 14.6-51.8 45.3 0 74.3 9 108.8 36 29-6.9 58.8-10 88.7-10 27 0 54.2 2.9 80.4 9.2 34-26.7 63-35.2 107.8-35.2 9.8 19.5 14.6 30.3 14.6 51.8 0 16.4-2.6 32.7-7.7 48.2 27.5 32.4 39 72.3 39 114.2zm-64.3 50.5c0-43.9-26.7-82.6-73.5-82.6-18.9 0-37 3.4-56 6-14.9 2.3-29.8 3.2-45.1 3.2-15.2 0-30.1-.9-45.1-3.2-18.7-2.6-37-6-56-6-46.8 0-73.5 38.7-73.5 82.6 0 87.8 80.4 101.3 150.4 101.3h48.2c70.3 0 150.6-13.4 150.6-101.3zm-82.6-55.1c-25.8 0-36.7 34.2-36.7 55.1s10.9 55.1 36.7 55.1 36.7-34.2 36.7-55.1-10.9-55.1-36.7-55.1z"/></svg>
  </div>
  <div class="md-source__repository">
    compbiozurich.github.io
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    General Information
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    News & Events
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../people" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Bioinformatics Groups
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4" >
        
          
          <label class="md-nav__link" for="__nav_4" id="__nav_4_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Lectures, Courses & Graduate Programs
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            Lectures, Courses & Graduate Programs
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../teaching" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Overview
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    
    
    
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_4_2" >
        
          
          <label class="md-nav__link" for="__nav_4_2" id="__nav_4_2_label" tabindex="0">
            
  
  <span class="md-ellipsis">
    Courses & Lectures
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_2_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_2">
            <span class="md-nav__icon md-icon"></span>
            Courses & Lectures
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/UZH-BIO390/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    UZH BIO390
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../teaching/UZH-BIO392/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    UZH BIO392
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Seminars
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_5" >
        
          
          <label class="md-nav__link" for="__nav_5" id="__nav_5_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Community Resources
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_5">
            <span class="md-nav__icon md-icon"></span>
            Community Resources
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../resources" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Tools & Resources
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../positions" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Positions
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                

  <h1 id="news-events-seminars">News, Events, Seminars ...</h1>
  
  

  

  
  
  
  
    
    
      
    
  
    
    
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
         
        
          
          
            
          
        
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  
    
    
         
        
      
    
  

<!--
First, upcoming events are processed with the "oldest" (i.e. closest in the
future) first if any exist
-->

  
  <h2 id="upcoming">Upcoming <a class="headerlink" href="#upcoming">&para;</a></h2>
    
      <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 05, 2023
      
    </span>
</aside>
<h3 class="title_news">TBA</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Samuel Miravet Verde (Sunagawa Lab ETHZ)</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> <mark>TBD</mark></p>
<a href="/news/2023-10-05-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
    
      <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 12, 2023
      
    </span>
</aside>
<h3 class="title_news">TBA</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> <mark>TBD</mark></p>
<a href="/news/2023-10-12-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
    
  

<!--
Then, previous events are processed with the latest first
-->

  <h2 id="previous">Previous <a class="headerlink" href="#previous">&para;</a></h2>

  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 28, 2023
      
    </span>
</aside>
<h3 class="title_news">Investigation of the potential of Covariates for Multiphenotype Studies (CMS) to improve genetic risk prediction</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Anja Estermann</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Polygenic risk scores (PRSs) estimate the genetic risk of an individual to develop a certain disease or trait and find application in disease prevention and personalized medicine. These scores are calculated based on summary statistics results of genome-wide association studies (GWAS). Covariates for multiphenotype studies (CMS) is an algorithm that has been developed to increase the detection of associated genetic variants in GWAS by leveraging covariates measured on behalf of a primary outcome.
<a href="/news/2023-09-28-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 14, 2023
      
    </span>
</aside>
<h3 class="title_news">SuperCellCyto: Enabling efficient analysis of large scale cytometry datasets</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Givanna Putri (WEHI)</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> The rapid advancements in cytometry technologies have enabled the quantification of up to 50 proteins across millions of cells at the single-cell resolution. The analysis of cytometry data necessitates the use of computational tools for tasks such as data integration, clustering, and dimensionality reduction. While numerous computational methods exist in the cytometry and single-cell RNA sequencing (scRNAseq) fields, many are hindered by extensive run times when processing large cytometry data containing millions of cells. Existing solutions, such as random subsampling, often prove inadequate as they risk excluding small, rare cell subsets.</p>
<p>To address this, we propose a practical strategy that builds on the SuperCell framework from the scRNAseq field. The supercell concept involves grouping single cells with highly similar transcriptomic profiles, and has been shown to be an effective unit of analysis for scRNAseq data.</p>
<p>We show that for cytometry datasets, there is no loss of information by grouping cells into supercells. Further, we demonstrate the effectiveness of our approach by conducting a series of downstream analyses on six publicly available cytometry datasets at the supercell level, and successfully replicating previous findings performed at the single cell level. We present a computationally efficient solution for transferring cell type labels from single-cell multiomics data which combines RNA with protein measurements, to a cytometry dataset, allowing for more precise cell type annotations.</p>
<p>Our SuperCellCyto R package and the associated analysis workflows are available on our GitHub repositories (<a href="https://github.com/phipsonlab/SuperCellCyto">github.com/phipsonlab/SuperCellCyto</a> and <a href="https://phipsonlab.github.io/SuperCellCyto-analysis/">phipsonlab.github.io/SuperCellCyto-analysis/</a>).</p>
<a href="/news/2023-09-14-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 15, 2023
      
    </span>
</aside>
<h3 class="title_news">splicekit: a comprehensive toolkit for splicing analysis from short-read RNA-seq</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Gregor Rot</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Splicing of RNA is a fundamental biological process. Dysregulation of splicing has been implicated in many human diseases and successfully exploited as a therapeutic target. Splicing analysis using short-read RNA-sequencing is a powerful technique to triage mechanism of action and safety profiles of drug candidates. There is, however, currently no comprehensive open-source software pipelines for such applications.
<a href="/news/2023-06-15-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 08, 2023
      
    </span>
</aside>
<h3 class="title_news">Modeling the Tumor Microenvironment with Graph Concept Learning</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Santiago Castro Dau</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Heterogeneity is an emergent property of tumors, linked to cancer resistance and poor treatment outcomes <sup id="fnref:1"><a class="footnote-ref" href="#fn:1">1</a></sup>. Geometric deep learning using graph representations has emerged as a promising approach to investigate tumor heterogeneity. Still, these approaches suffer from interpretability and transferability limitations <sup id="fnref:2"><a class="footnote-ref" href="#fn:2">2</a></sup>. In this work, we propose a geometric deep-learning model with an interpretability framework to predict metadata from spatial datasets.
<a href="/news/2023-06-08-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 20, 2023
      
    </span>
</aside>
<h3 class="title_news">Microbiome Signatures in Host - Spatial and Single Cell Transcriptomics</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Stine Anzböck</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> The tumor-associated microbiome is an integral part of the tumor microenvironment and holds great promise for a better understanding of carcinogenesis, response to therapy and early diagnosis of cancer. However, current methods limit the understanding of the spatial distribution of the microbiome in the tumor and their interaction with host cells. To this end, we have developed SpaceMicrobe, a novel computational framework to detect and profile the microbiome in 10X Visium Spatial Gene Expression data, a widely used commercially-available spatial transcriptomics technology.
<a href="/news/2023-04-20-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 13, 2023
      
    </span>
</aside>
<h3 class="title_news">Variational Autoencoders Supporting Conditioning in Single Cell Transcriptomics and Their Consistency</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Eljas Röllin</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Prediction of cell response to perturbation is a key goal in computational biology. To this end, different Variational Autencoder (VAE) based models have been suggested in the past. We are interested in such models and their ability to recognize their own output. We argue that this is a desirable model property, and phrase it in the context of cycle consistency.
<a href="/news/2023-04-13-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 23, 2023
      
    </span>
</aside>
<h3 class="title_news">Analysis of copy number variant heterogeneity in the hierarchical NCIt cancer classification system</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Ziying Yang</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Cancers are heterogeneous diseases with unifying features of abnormal and consuming cell growth, where the deregulation of normal cellular functions is caused by accumulative mutations. Due to these mutations, malignant tumors present with patterns of somatic genome variants on diverse levels of heterogeneity. Among the different mutation types, genomic copy number aberrations(CNA) have emerged as one of the most distinct classes.
<a href="/news/2023-03-23-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 09, 2023
      
    </span>
</aside>
<h3 class="title_news">Detecting Single Cell Blasts in Acute Myeloid Leukaemia using an Auto-Encoder</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Alice Driessen</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel <strong>Y13-K-05</strong> and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Acute myeloid leukaemia (AML) is a haematological cancer in the bone marrow, with accumulation and expansion of immature cells of the myeloid lineage. Unfortunately, almost half of paediatric AML patients relapse after standard treatment with chemotherapy or stem cell transplantation. Personalised medicine including immunotherapies have the potential to target chemotherapy resistant cells and achieve long-term remission. However, identifying suitable targets for AML therapy is hampered by high patient heterogeneity, complex disease evolution and challenging discrimination between aberrant and developing cells. Therefore, we aimed to build a single-cell cytometry AML map to identify malignant cells and place them along the developmental trajectory using data from 20 patients and three time points over the course of the disease.
<a href="/news/2023-03-09-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 16, 2023
      
    </span>
</aside>
<h3 class="title_news">Multi-omics studies of cancer signalling and immune infiltration</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Pedro Beltrao</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Genetic alterations in cancer cells trigger oncogenic transformation, a process largely mediated by the dysregulation of kinase and transcription factor (TF) activities. While the mutational profiles of thousands of tumours have been extensively characterised, the measurements of protein activities have been technically limited until recently. We compiled public data of matched genomics and (phospho)proteomics measurements for 1,110 tumours that we used to estimate activity changes in 218 kinases and 292 TFs.
<a href="/news/2023-02-16-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 26, 2023
      
    </span>
</aside>
<h3 class="title_news">Inflammatory bowel disease at single-cell and sub-cellular spatial resolution</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Helena L. Cromwell</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> CosMx Spatially Molecular Imaging (SMI) (NanoString) is a recently developed technology that enables spatially resolved profiling of tissue at molecule-level resolution at an unprecedented scale (up to 1,000 RNA analytes). We applied scRNA-seq and CosMx SMI to investigate the molecular basis of Ulcerative colitis and Crohn’s disease: chronic inflammatory bowel diseases (IBD) that show a perplexing heterogeneity in manifestations and response to treatment.
<a href="/news/2023-01-26-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 24, 2022
      
    </span>
</aside>
<h3 class="title_news">Searching in nucleotide archives at Petabase scale with MetaGraph</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Mikhail Karasikov</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> High-throughput sequencing data is continuously accumulating in massive archives such as the NCBI Sequence Read Archive (SRA), which currently contains over 50 Petabytes of sequences. However, despite recent advances, even such a basic operation as sequence search effectively remains intractable due to the lack of cost-efficient solutions. To address this problem and enable aggregated data analysis at Petabase scale, we developed MetaGraph, a tool for indexing very large collections of sequences in de Bruijn graphs.
<a href="/news/2022/2022-11-24-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 10, 2022
      
    </span>
</aside>
<h3 class="title_news">Nucleosome footprints in the cell-free DNA of cancer patients</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Zsolt Balázs</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Cell-free DNA (cfDNA) is free floating DNA released from cells into bodily fluids like blood. cfDNA sequencing is being increasingly utilized in cancer diagnosis and monitoring. While many applications focus on detecting cancer-specific mutations, cfDNA can potentially tell us much more about the disease. Nucleosome-protected regions are slow to degrade in the bloodstream, therefore, nucleosome occupancy in the cell of origin can be inferred from cfDNA fragmentation. 
<a href="/news/2022/2022-11-10-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 13, 2022
      
    </span>
</aside>
<h3 class="title_news">Genome-wide study of variations in Plasmodium falciparum and their association with different malaria interventions in Tanzania</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Catherine Bakari Mvaa (Christian Nsanzabana group @ TPH)</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Malaria affects millions of people globally, and it remains one of the major public health problems occurring in most parts of Tanzania, causing thousands of deaths each year, especially among vulnerable persons in rural settings with poor health systems. Even though the burden has decreased due to the deployment of different interventions in recent years, all those gains are threatened by biological threats that largely affect the success of different control strategies. The parasite uses different mechanisms to escape diagnostic tools and drug treatment, challenging one of the key control strategies, based on prompt detection and case management. Sequencing technologies, especially Next Generation Sequencing (NGS) have played a major role in understanding parasite genomic variation including drug resistance, diagnostic resistance, and population structure.
<a href="/news/2022/2022-10-13-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 11, 2022
      
    </span>
</aside>
<h3 class="title_news">A journey into the nucleus: <b>Lnc-ing</b> paraspeckles, lncRNA, RNA processing and phase separation</h3>
      
<h5 class="title_news">NCCR RNA and Disease seminar</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h4 id="archa-fox-the-university-of-western-australia">Archa Fox, The University of Western Australia</h4>
<h5 id="host-prof-dr-magdalini-polymenidou">Host: Prof. Dr. Magdalini Polymenidou</h5>
<p>The seminar will be preceded by a short presentation:</p>
<h3 id="unraveling-genetic-modifiers-and-cellular-pathways-of-physiological-tdp-43-oligomerization">Unraveling genetic modifiers and cellular pathways of physiological TDP-43 oligomerization.”</h3>
<h4 id="laura-de-vos-polymenidou-group-uzh">Laura de Vos (Polymenidou group, UZH)</h4>

<a href="/news/2022/2022-10-11-Archa-Fox-NCCR-RNA/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 15, 2022
      
    </span>
</aside>
<h3 class="title_news">Comprehensive network prediction for any fully sequence genome in STRING database</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Damian Szklarczyk</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <p><strong>Abstract</strong> Every day new genomes are sequenced and existing genomes are re-sequenced and re-annotated. In the new version of STRING the user can submit any fully sequenced genome for complete network and functional annotation. To do so. will require only minimal input from the user in a form of the proteome in a FASTA format and, if known, a taxonomical clade of the given genome.
<a href="/news/2022/2022-09-15-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 06, 2022
      
    </span>
</aside>
<h3 class="title_news">Towards a quantitative understanding of long-range transcriptional regulation</h3>
      
<h5 class="title_news">ETHZ seminar - Luca Giorgetti, FMI Basel</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>recently published <a href="https://www.nature.com/articles/s41586-022-04570-y">Nature paper</a> on “Nonlinear control of transcription through enhancer–promoter interactions”</li>
</ul>

<a href="/news/2022/2022-09-06-Luca-Giorgetti-ETHZ/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 18, 2022
      
    </span>
</aside>
<h3 class="title_news">CompbioZurich Website Redesign</h3>
      
<h5 class="title_news">Re-implementing the Site with Mkdocs</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <p>The <a href="http://compbiozurich.org">compbiozurich.org</a> website has been redesigned
based on the Mkdocs framework &amp; using the <a href="https://squidfunk.github.io/mkdocs-material/">Material</a> template system.
<a href="/news/2022/2022-07-18-website-redesign/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 16, 2022
      
    </span>
</aside>
<h3 class="title_news">Novel Minimal HDV-like Ribozymes</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Lukas Malfertheiner</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <p><strong>Abstract</strong> The hepatitis delta virus (HDV) ribozyme catalyzes site-specific self-cleavage and was first discovered in the single-stranded circular RNA virus HDV. HDV-like ribozymes (DRZ) share the conserved nested double-pseudoknot structure motif. Through a bioinformatic search using an adapted minimal active DRZ motif, we discovered hundreds of novel minimal DRZ sequences in bacteriophage genomes associated with the human microbiome. A subclass of these hits was identified to occur in direct conjunction with viral tRNA genes, indicating that we have found a solely RNA-based factor that can site-specifically cleave tRNA 3'-trailers, thus making large protein enzymes redundant for this task. </p>
<ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<a href="/news/2022/2022-06-16-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 19, 2022
      
    </span>
</aside>
<h3 class="title_news">Supervised spatial inference of dissociated single-cell data with _SageNet_</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Elyas Heidari</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Spatially-resolved transcriptomics uncovers patterns of gene expression at supercellular, cellular, or subcellular resolution, providing insights into spatially variable cellular functions, diffusible morphogens, and cell-cell interactions. However, for practical reasons, multiplexed single cell RNA-sequencing remains the most widely used technology for profiling transcriptomes of single cells, especially in the context of large-scale anatomical atlassing.
<a href="/news/2022/2022-05-19-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 12, 2022
      
    </span>
</aside>
<h3 class="title_news">The resemblance is uncanny a.k.a how similar are cancer cell lines really to their origins?</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Rahel Paloots</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p>Cancer cell lines are good and inexpensive models to study disease mechanisms and identify possible new drugs, However, many times cancer cell lines have been found to be either misidentified or contaminated. Additionally, cancer cell lines only represent a small clonal population of the origin as well as accumulate some additional mutations during in vitro handling. 
<a href="/news/2022/2022-05-12-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 05, 2022
      
    </span>
</aside>
<h3 class="title_news">Copy number variation data calibration towards intgerative analysis in cancer</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Hangjia Zhao</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Genomic instability is common in human cancer. As a form of genomic instability, copy number variations (CNV) play an important role in cancer development. Elucidating the relationship between CNV and cancer evolvement can improve the understanding of the pathogenetic mechanism of cancer.</p>

<a href="/news/2022/2022-05-05-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 24, 2022
      
    </span>
</aside>
<h3 class="title_news">Frequent co-regulation of splicing and polyadenylation by RNA-binding proteins inferred with MAPP</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Maciek Bak | Biozentrum, University of Basel | Swiss Institute of Bioinformatics</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p>The processing of nascent pre-mRNA consists of many steps, where splicing and primary transcript polyadenylation play key roles in determining transcriptome and subsequently, proteome diversity. Several studies indicate that many RNA-binding proteins (RBPs) act both on splicing as well as 3’ end processing but the context of this multi-level regulation and the full spectrum of RBPs involved are yet to be discovered. To facilitate answering these questions we have developed a novel computational method to identify RBPs that could shape the pre-mRNA maturation.</p>

<a href="/news/2022/2022-03-24-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 06, 2022
      
    </span>
</aside>
<h3 class="title_news">An ion channel that allows you to see at night: the cryo-EM structure of the rod CNG channel opens up the hypothesis of mRNA editing on the CNGB1 sequence</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Jacopo Marino, Laboratory of Biomolecular Research, Paul Scherrer Institut</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> I will present the latest advances in cryo-electron microscopy and how this technique allows to solve protein structures at fast pace also of difficult protein targets, thus revolutionizing the field of structural biology and biomedicine. I will then focus on the recent developments on solving the structure of a ion channel that is located in rod photoreceptors of the retina, a protein complex that was discovered nearly 35 years ago and its structure has finally being revealed (<a href="http://dx.doi.org/10.1038/s41594-021-00700-8">article</a>).</p>

<a href="/news/2022/2022-01-06-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 16, 2021
      
    </span>
</aside>
<h3 class="title_news">Taxonomic profiling of metagenomes from diverse environments with mOTUs3</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Hans-Joachim Ruschewey</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Taxonomic profiling is a crucial task in microbiome research that aims at detecting and quantifying the relative abundance of microbes in biological samples. However, in many environments, varying fractions of microbial species still lack a sequenced genome and remain unaccounted for during taxonomic profiling based on shotgun metagenomic data.</p>

<a href="/news/2021/2021-12-16-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 09, 2021
      
    </span>
</aside>
<h3 class="title_news">mVIRs: A tool to identify and locate inducible prophages in microbial genomes using NGS data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Mirjam Zünd</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> An estimated 40 - 50% of mammalian gut bacteria carry a prophage in their genome. Prophages can be induced upon stress and produce phage particles to infect and kill bacterial populations and thereby directly modulate the composition of the gut microbiota. Despite advances in studying prophages using whole-genome sequencing, challenges in identifying and locating inducible prophages within their host have limited our understanding of phage-bacterial interactions. </p>

<a href="/news/2021/2021-12-09-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 02, 2021
      
    </span>
</aside>
<h3 class="title_news">CanIsoNet beta-version: A Database to Study the Functional Impact of Isoform Switching Events in Cancer</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Tülay Karakulak</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p>Alternative splicing, as an essential regulatory mechanism in normal mammalian cells, is frequently disturbed in cancer. Switches in the expression of alternative isoforms can alter protein interaction networks of associated genes giving rise to cancer progression and metastases. We have recently analyzed the pathogenic impact of switching events in 1209 cancer samples covering 27 different cancer types.</p>

<a href="/news/2021/2021-12-02-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 18, 2021
      
    </span>
</aside>
<h3 class="title_news">Structured metadata for genomic correlations in the Progenetix database</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Ziying Yang</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p><img alt="Progenetix logo" src="https://progenetix.org/img/progenetix-logo-black.png" style="float: right; width: 200px; margin-top: -80px;" /><strong>Abstract</strong> Enormous amounts of biomedical data have been and are being produced at an unprecedented rate by researchers all over the world. However, in order to enable reuse, there is an urgent need to understand the structure of datasets, the experimental conditions under which they were produced and the information that other investigators may need to make sense of the data.</p>

<a href="/news/2021/2021-11-18-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 28, 2021
      
    </span>
</aside>
<h3 class="title_news">Functional implications of Short tandem repeat (STR) variation in NGS data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Max Verbiest</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p>Short tandem repeats (STRs) are genomic elements that consist of consecutive repetitions of a 1-6 nucleotide motif. They are abundant in the human genome and are known to be mutational hotspots. Several cancer types are known to have a microsatellite instability high (MSI-H) subtype wherein the DNA mismatch repair system is defective, leading to hypermutation of STRs.</p>

<a href="/news/2021/2021-10-28-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 21, 2021
      
    </span>
</aside>
<h3 class="title_news">Identifying clones and quantifying diversity in repertoire sequencing data.</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Siyuan Luo</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel Y55-l-06/08 and ZOOM Call</li>
</ul>
<p>The adaptive immune system is remarkable for its ability to produce immunoglobulins that can specifically bind to a wide variety of antigens. During adaptive immune responses, activated B cells expand and undergo accelerated mutation of their B cell receptor (BCR), forming a clone of diversified cells that can be related back to a common ancestor.</p>

<a href="/news/2021/2021-10-21-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 14, 2021
      
    </span>
</aside>
<h3 class="title_news">Supervised Hierarchical Autoencoders for Multi-Omics Integration in Cancer Survival Models</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - David Wissel</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:00 UZH Irchel Y55-l-06/08 and ZOOM Call (12:15)</li>
</ul>
<p><strong>Abstract</strong> With the increasing amount of high-throughput sequencing data becoming available, the proper integration of differently sized and heterogeneous molecular and clinical groups of variables has become crucial in cancer survival models. </p>

<a href="/news/2021/2021-10-14-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 07, 2021
      
    </span>
</aside>
<h3 class="title_news">Comparison of Probabilistic Cell Type Annotation Methods Applied to Immunological CITE-seq Data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Ahmad Al Ajami</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel <strong>Y34-K-01</strong> and ZOOM Call</li>
</ul>
<p>Abstract: A fundamental task in the analysis of single-cell RNA-sequencing data is cell type annotation. Current cell type annotation methods rely on either prior biological knowledge, which is not always available in a comprehensive and consistent manner, or well-annotated reference datasets, which are not always complete, i.e., not all cell types present in the target are represented in the reference data. We propose conducting cell type annotation in a probabilistic framework to fill the gap incomplete references create.
<a href="/news/2021/2021-10-07-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 30, 2021
      
    </span>
</aside>
<h3 class="title_news">A global exploration of factors contributing to rates of horizontal gene transfer</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Marija Dmitrijeva</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 UZH Irchel <strong>Y34-K-01</strong> and ZOOM Call</li>
</ul>
<p>Horizontal gene transfer - the exchange of genetic material between organisms through mechanisms other than reproduction, has been shown to play an important role in prokaryotic genome evolution. Previous studies have predominantly focused on recent horizontal gene transfer events, often limiting investigation to human-associated microorganisms. In this study, we aim to conduct a more global survey of the factors contributing to horizontal gene transfer.
<a href="/news/2021/2021-09-30-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 17, 2021
      
    </span>
</aside>
<h3 class="title_news">Introducing Progenetix Cell Line Beacon - A comprehensive cancer cell line variant knowledge resource</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Rahel Paloots</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p>Cancer cell lines are good models for studying the disease mechanisms and testing for possible drugs. For the cell lines to be accurate representations of the disease, they would need to be genetically highly similar to their primary neoplasias. Another issue when working with cell lines is the possible contamination or misidentification of the cell lines. 
To address both of these concerns, the genetics of both cancer cell lines and their origins will need to be evaluated. Cancers as well as cancer cell lines exhibit copy number variation (CNV) profiles that show regions in the chromosomes that have been deleted or amplified.
<a href="/news/2021/2021-06-17-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 27, 2021
      
    </span>
</aside>
<h3 class="title_news">A computational framework to analyse phenotypic heterogeneity in spatial single cell data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Adriano Martinelli</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <p>Highly-multiplexed imaging technologies with single-cell resolution, such as imaging mass cytometry or multiplexed ion beam imaging, are revolutionising the field of biomedical research.</p>

<a href="/news/2021/2021-05-27-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 20, 2021
      
    </span>
</aside>
<h3 class="title_news">Stratifying sources of heterogeneity in single-cell whole-genome bisulfite sequencing datasets</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Emanuel Sonder</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <p>DNA methylation is on the base level a binary mark. However, when calculating average methylation rates across cells in bulk samples, certain regions of the genome exhibit intermediate methylation values, indicating heterogeneous methylation patterns.</p>

<a href="/news/2021/2021-05-20-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 06, 2021
      
    </span>
</aside>
<h3 class="title_news">A morphometric framework for the embryo-wide quantification of tissue organisation at single cell resolution</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Max Brambach</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>

<a href="/news/2021/2021-05-06-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 25, 2021
      
    </span>
</aside>
<h3 class="title_news">Discovering copy number variation across multiple cancer types</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Qingyao Huang</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> Genomic variations are direct cause of tumor formation and accomplice in its continuous evolution. While point mutations can be pinpointed to a targeted genetic element, copy number variations (CNVs) involve copy number gain or loss of a large DNA segment which often covers hundreds of genetic elements in one event. Although the vast majority of variations are not directly fucntionally cancer promoting, we observe consistency in CNV landscape within the same cancer types and corresponding increase in heterogeneity along with increased distinction in physiology and morphology. This implies that particular CNV may promote cancer type-specific progression.
<a href="/news/2021/2021-03-25-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 18, 2021
      
    </span>
</aside>
<h3 class="title_news">Analysis and Method development for High-Throughput DNA Methylation and Gene Expression with Applications to Colorectal Tumors</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Stephany Orjuela</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>

<a href="/news/2021/2021-03-18-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 11, 2021
      
    </span>
</aside>
<h3 class="title_news">Degradation of human mRNA transcripts over time as an indicator of the time since deposition (TsD) in biological crime scene traces</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Giancarlo Russo</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p><em>Abstract</em> Knowledge about the age of a stain or rather the time since deposition (TsD) would provide law enforcing authorities with valuable information for the prosecution of criminal offences. There is no reliable method for estimation of TsD available yet. The aim of this study is to gain further insight into the RNA degradation pattern of forensically relevant body fluids, and to find markers that allow to estimate the TsD interval. Stain samples from several body fluids were exposed to indoora nd outdoor conditions for up to 1.5 years. We were able to identify degradation as well as body fluid specific signatures in both indoor and outdoor samples.
<a href="/news/2021/2021-03-11--zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 04, 2021
      
    </span>
</aside>
<h3 class="title_news">Leveraging single-cell RNA-seq with CITE-seq data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Anthony Sonrel</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p>Recently, the ability to profile gene expression at the single-cell level has been expanded to include the measure of both RNA and protein levels from the same cells (<a href="https://www.nature.com/articles/nmeth.4380">Stoeckius et al., 2017</a>). While RNA-based data has broadly proven sufficient for cell type detection and classification, its usage for determining drug target is still limiting: we still often rely on the assumption that RNA expression levels are representative of protein expression levels.
<a href="/news/2021/2021-03-04-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      February 04, 2021
      
    </span>
</aside>
<h3 class="title_news">Protein-protein interaction prediction using co-evolution based method</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Tao Fang</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p>It has been found that protein families  consisting of  orthologous proteins  usually show  divergent  sequence similarities due to amino acid mutations in evolutionary history while structures of   protein families  are usually  much more conserved. This phenomenon can be explained by two types of mutant positions in the protein sequence. One is conserved  regions across the protein family  and  even a slight change in this region could result in critical loss of protein structure or function.  Another is the so-called compensated mutation or coevolution region in protein.
<a href="/news/2021/2021-02-04-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 21, 2021
      
    </span>
</aside>
<h3 class="title_news">From Biomolecular Networks to Pathways: Tools for the Interpretation of Genome-Wide Experiments</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Annika Gable</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p>Functional enrichment analysis (also called gene set enrichment analysis or pathway analysis) is a widely-used method to interpret genome-wide experiments, especially in transcriptomics and proteomics. The method relies on previously annotated gene sets, such as pathways, cellular locations, protein families, disease genes, or other annotation terms.
<a href="/news/2021/2021-01-21-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 10, 2020
      
    </span>
</aside>
<h3 class="title_news">Phylogeny and Metabolic Potential of SAR324</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Lukas Malfertheiner</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p>The bacterial SAR324 cluster is ubiquitous in the ocean, especially around hydrothermal vents and in the deep sea, where it can account for up to 20% of the whole bacterial community. According to a new taxonomy generated using multiple universal protein-coding genes instead of the previously used 16s rRNA single gene marker, the former Deltaproteobacteria cluster SAR324 is since 2018 classified as its own phylum. Yet, very little is known about the phylogeny, metabolic potential and the influence of SAR324 on the different environments where they live. We downloaded all publicly available SAR324 genomes (65) from the genome taxonomy database, and also constructed 18 new genomes out of different oceanic metagenomic samples, including from under the Ross Ice Shelf in the Antarctica.
<a href="/news/2020/2020-12-10-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 19, 2020
      
    </span>
</aside>
<h3 class="title_news">Doublet identification and characterization in single-cell data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Pierre-Luc Germain</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 om  ZOOM Call</li>
</ul>
<p>Doublets (i.e. two cells captured as a single cell) can form at a high frequency in single-cell sequencing studies (often 10-20%). They can appear as spurious 'new cell types' or distort trajectory and co-expression analyses. While experimental strategies can be used to mitigate this effect, they are insufficient and need to be complemented with doublet identification methods.
<a href="/news/2020/2020-11-19-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 05, 2020
      
    </span>
</aside>
<h3 class="title_news">2020 11 05 zh seminars in bioinformatics</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - João Rodrigues</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h3 id="modelling-and-estimating-interactions-from-time-series-observations-of-synthetic-microbial-community-experiments">Modelling and estimating interactions from time-series observations of synthetic microbial community experiments</h3>
<ul>
<li>12:15 om  ZOOM Call</li>
</ul>

<a href="/news/2020/2020-11-05-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      September 03, 2020
      
    </span>
</aside>
<h3 class="title_news">ReSeq simulates realistic Illumina high-throughput sequencing data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Stephan Schmeing</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:00 in Y-55L-06, UZH Irchel Campus and 12:15 om  ZOOM Call</li>
</ul>
<p><strong>Abstract</strong> In high-throughput sequencing data, performance comparisons between computational tools are essential for making informed decisions in the data processing from raw data to the scientific result. Simulations are a critical part of method comparisons, but for standard Illumina sequencing of genomic DNA, they are often oversimplified, which leads to optimistic results for most tools.
<a href="/news/2020/2020-09-03-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 09, 2020
      
    </span>
</aside>
<h3 class="title_news">2020 07 09 zh seminars in bioinformatics</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Lennart Opitz</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h3 id="hidra-seq-high-throughput-sars-cov-2-detection-by-rna-barcoding-and-amplicon-sequencing">HiDRA-seq: High-Throughput SARS-CoV-2 Detection by RNA Barcoding and Amplicon Sequencing</h3>
<ul>
<li>12:00 in Y-55L-06, UZH Irchel Campus and 12:15 om  ZOOM Call</li>
</ul>

<a href="/news/2020/2020-07-09-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 25, 2020
      
    </span>
</aside>
<h3 class="title_news">Flanking sequence preference modulates _de novo_ DNA methylation in the mouse genome</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Izaskun Mallona</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 (ZOOM Call)</li>
</ul>
<p><strong>Abstract</strong> Mammalian de novo DNA methyltransferases (DNMT) are responsible for the establishment of cell-type-specific DNA methylation in healthy and diseased tissues. Through genome-wide analysis of de novo methylation activity in murine stem cells we uncover that DNMT3A prefers to methylate CpGs followed by cytosines or thymines, while DNMT3B predominantly methylates CpGs followed by guanines or adenines. These signatures are further observed at non-CpG sites, resembling methylation context observed in specialised cell types, including neurons and oocytes.
<a href="/news/2020/2020-06-25-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 18, 2020
      
    </span>
</aside>
<h3 class="title_news">Progenetix data acquisition & processing pipeline</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Qingyao Huang</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 (ZOOM Call)</li>
</ul>
<p><strong>Abstract</strong> Over the years we maintain and update the Progenetix database, which provides an overview of copy number abnormalities in human cancer from currently 113322 array and chromosomal Comparative Genomic Hybridization (CGH) experiments, as well as Whole Genome or Whole Exome Sequencing (WGS, WES) studies. The cancer profile data in Progenetix was curated from 1600 articles and represents 420 and 542 different cancer types, according to the International classification of Diseases in Oncology (ICD-O) and NCIt "neoplasm" classification, respectively.
In this seminar, I will give an overview of our current pipeline from data retrieval, metadata extraction/curation, data processing and evaluation to the inclusion/exlusion in the database.
<a href="/news/2020/2020-06-18-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 04, 2020
      
    </span>
</aside>
<h3 class="title_news">distinct: a method for differential analyses via hierarchical permutation tests, with applications to single-cell data</h3>
      
<h5 class="title_news">Zurich Seminars in Bioinformatics - Simone Tiberi</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 (ZOOM Call)</li>
</ul>

<a href="/news/2020/2020-06-04-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      May 28, 2020
      
    </span>
</aside>
<h3 class="title_news">aGOtool.org web-tool for protein-centric functional enrichment</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - David Lyon</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 (ZOOM Call)</li>
</ul>

<a href="/news/2020/2020-05-28-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 16, 2020
      
    </span>
</aside>
<h3 class="title_news">Setting up a pipeline for horizontal gene transfer detection</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Marija Dmitrijeva</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 (Skype Call)</li>
</ul>
<p><strong>Abstract</strong> In addition to vertically transferring genes to their descendants, bacteria can horizontally transfer genes to other members in their community. These horizontally transferred genes frequently confer various selective advantages to the recipient such as antibiotic resistance and the ability to scavenge rare nutrients. Thus, gaining insight into the principles behind horizontal gene transfer is important for understanding the evolution of prokaryotic genomes.
<a href="/news/2020/2020-04-16-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 09, 2020
      
    </span>
</aside>
<h3 class="title_news">Censored predictors in single cell regression models</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Reto Gerber</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 (Skype Call)</li>
</ul>
<p><strong>Abstract</strong> Flow and mass cytometry enable the classification of a large number of cells into cell populations. Furthermore, the abundance of cell populations can have an association with a covariate that has additional information about the sample, such as treatment group, age, sex, survival time, etc. In differential abundance analysis, the goal is then to find cell populations whose abundance changes according to a covariate.
<a href="/news/2020/2020-04-09-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 19, 2020
      
    </span>
</aside>
<h3 class="title_news">The Microbe Atlas Project</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Joao Rodrigues</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 (Skye call)</li>
</ul>

<a href="/news/2020/2020-03-19-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 12, 2020
      
    </span>
</aside>
<h3 class="title_news">HackyHour Zurich - data analysis made social</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Nils Eling</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15, Y34-J-01, UZH Irchel campus</li>
</ul>
<p><strong>Abstract</strong> HackyHour is an internationally recognized effort to form local communities of students and researchers and offer a space to discuss ideas, problems and new tools related to scientific data analysis. Key features of such a recurring event are the social and informal setting, an inclusive environment inviting beginners and experts and an interactive exchange of computational knowledge.
<a href="/news/2020/2020-03-12-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 11, 2019
      
    </span>
</aside>
<h3 class="title_news">Knowledge standards drive clinical interpretation of genomic variants in cancers</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Alex H. Wagner</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h4 id="washington-university-school-of-medicine-st-louis">Washington University School of Medicine, St. Louis</h4>
<ul>
<li>12:15 in Y-55L-12, UZH Irchel campus (room subject to change)</li>
</ul>
<p><strong>Abstract</strong> The use of diagnostic gene sequencing in the clinical setting is increasingly common. Established sequencing panels targeting genes involved in cancer predisposition, tumor growth, and chemoresistance are widely available, and numerous efforts to validate whole-exome and whole-genome clinical sequencing assays are underway. As a result, genome analysts and molecular pathologists are increasingly tasked to the labor-intensive process of interpreting the clinical significance of tumor variants. Building these clinical reports entails reviewing the literature, population frequency databases, and predicted functional impact of each observed variant.
<a href="/news/2019/2019-12-11-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      December 04, 2019
      
    </span>
</aside>
<h3 class="title_news">Interval statistics reveal frequently altered gene copies across cancers</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Qingyao Huang</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>Wednesday, 2019-12-04</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>
<p><strong>Abstract</strong> Various types of genomic aberration are observed in malignant tumours, including single nucleotide mutation and copy number variations (CNV). While the former are focal changes, the effects of which can be localised to individual genetic elements, the latter often contain megabases of genomic regions, covering hundreds of genetic elements. CNV has been successfully used on a coarse scale (cytoband level) as molecular features to define subtypes in many cancer entities. However, a systematic understanding of the CNV patterns in cancers is not established.
<a href="/news/2019/2019-12-04-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 27, 2019
      
    </span>
</aside>
<h3 class="title_news">2019 11 27 zh seminars in bioinformatics</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Joao Rodrigues</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>

<a href="/news/2019/2019-11-27-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      November 13, 2019
      
    </span>
</aside>
<h3 class="title_news">Functional enrichment in the STRING protein-protein interaction database</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Annika Gable</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <p><em>Abstract</em> I will talk about the extended enrichment functionality in STRING, which accepts whole genomes or proteomes with values, including RNASeq, microarrays, ranked genomes, protein abundances, etc. 
A high-powered enrichment of various annotation databases such as GO-terms is computed within a minute.</p>
<p>Also, I will talk about what we can learn from analyzing the users’ inputs in an aggregated fashion, as well as providing a preliminary comparison of enrichment results between our new enrichment and a similar tool, GSEA.</p>
<a href="/news/2019/2019-11-13-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 09, 2019
      
    </span>
</aside>
<h3 class="title_news">2019 10 09 zh seminars in bioinformatics</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Anna Quaglieri</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h5 id="walter-and-eliza-hall-institute-of-medical-research-melbourne">Walter and Eliza Hall Institute of Medical Research, Melbourne</h5>
<h3 id="using-the-transcriptome-to-study-therapy-response-in-leukaemia">Using the transcriptome to study therapy response in leukaemia</h3>
<ul>
<li>Wednesday, 2019-10-09</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>

<a href="/news/2019/2019-10-09-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      October 02, 2019
      
    </span>
</aside>
<h3 class="title_news">High-dimensional flow mass cytometry with Imaging Mass Cytometry in WNV and ZIKV encephalitis</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Tom Ashhurst | Sydney Cytometry Facility, The University of Sydney and Centenary Institute</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>Wednesday, 2019-10-02</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>
<p><em>Abstract</em> West Nile virus (WNV) and Zika virus (ZIKV) are responsible for 
central nervous system (CNS) diseases including encephalitis, Guillian-Barré 
syndrome and congenital neurological abnormalities in the developing fetus. 
A successful immune response must carefully balance pathogen elimination against 
destruction of the host’s CNS. To study these diseases in detail, we analyzed 
the cellular composition of virally infected mice tissue by high-dimensional 
flow (30-parameter) and mass (47-parameter) cytometry along with Imaging 
Mass Cytometry™ and characterized and contrasted the evolving immune response 
over time.
<a href="/news/2019/2019-10-02-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      July 21, 2019
      
    </span>
</aside>
<h3 class="title_news">ISMB - ECCB 2019</h3>
      
<!-- Use a hidden p tag to provide a preview -->
      <p>ISMB/ECCB 2019 is the largest and most high profile annual meeting of scientists working in computational biology. This year's conference will be on July 21-25 in Basel.</p>

<a href="/news/2019/2019-07-21-ISMB-ECCB-2019/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 05, 2019
      
    </span>
</aside>
<h3 class="title_news">On the discovery of population-specific state transitions from single-cell RNA sequencing data</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Helena L. Crowell</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>Wednesday, 2019-06-05</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>
<p><em>Abstract</em> Single-cell RNA-sequencing (scRNA-seq) has quickly become an empowering technology to profile the transcriptomes of individual cells on a large scale. Many early analyses of differential expression have aimed at identifying differences between cell types (or clusters), and thus are focused on finding markers for cell populations (experimental units are cells).
<a href="/news/2019/2019-06-05-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      June 04, 2019
      
    </span>
</aside>
<h3 class="title_news">Wrangling of copy number data in cancer genomic analysis</h3>
      
<h5 class="title_news">Seminar in Interdisciplinary Life Science (SILS)</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h4 id="michael-baudis-bo-gao">Michael Baudis &amp; Bo Gao</h4>
<ul>
<li>Tuesday, 2019-06-04</li>
<li>17:30 in Y03-G85, UZH Irchel campus</li>
</ul>

<a href="/news/2019/2019-06-04-SILS/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 17, 2019
      
    </span>
</aside>
<h3 class="title_news">2019 04 17 zh seminars in bioinformatics</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Bo Gao</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h3 id="minimum-error-normalization-for-copy-number-analysis">Minimum Error Normalization for Copy Number Analysis</h3>
<ul>
<li>Wednesday, 2019-04-17</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>

<a href="/news/2019/2019-04-17-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 16, 2019
      
    </span>
</aside>
<h3 class="title_news">Novel in  silico  methods  to  unveil  DNA  methylation  heterogeneity  in  cohorts,  tissues  and  cells</h3>
      
<h5 class="title_news">UZH DMMD Seminar</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h4 id="izaskun-mallon">Izaskun Mallon</h4>
<ul>
<li>Tuesday, 2019-04-16</li>
<li>14:00 in Y15-G-40, UZH Irchel campus</li>
</ul>
<p><em>Abstract</em> Epigenomics modulates cell development and function, and its alterations capture and mediate functional properties of cells, tissues and states, including differentiation and disease. From all epigenomics marks, DNA methylation can be measured reliably in almost any sample type, including clinical specimens.
<a href="/news/2019/2019-04-16-izaskun_mallona_dmmd_seminar/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 10, 2019
      
    </span>
</aside>
<h3 class="title_news">2019 04 10 zh seminars in bioinformatics</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Marija Dmitrijeva</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <h3 id="dynamics-of-the-lung-microbiome-in-patients-with-cystic-fibrosis">Dynamics of the Lung Microbiome in Patients with Cystic Fibrosis</h3>
<ul>
<li>Wednesday, 2019-04-10</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>

<a href="/news/2019/2019-04-10-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      April 03, 2019
      
    </span>
</aside>
<h3 class="title_news">Polyploid crop genome assembly by optical genome mapping and homeolog specific genomics analysis</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Masaomi Hatakeyama</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>Wednesday, 2019-04-03</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>
<p><em>Abstract</em> I will show the latest update of Finger millet genome assembly
scaffolded by Bionano Saphyr at chromosomal length scale. I will
introduce yet another polyploid analysing tool called EAGLE-RC and
show some results on the resequencing data.</p>
<a href="/news/2019/2019-04-03-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 25, 2019
      
    </span>
</aside>
<h3 class="title_news">Relaunch - compbiozurich.org with Github Based Hosting</h3>
      
<!-- Use a hidden p tag to provide a preview -->
      <p>As of 2019-03-25, the <em>compbiozurich.org</em> webste is being hosted as a Github project.
This move will allow an "open contributions" structure, in which members of Zurich's Bioinformatics &amp; Computational Biology community can participate in content creation.
Typical examples of such user-driven updates could be:</p>
<ul>
<li>seminar announcements</li>
<li>links to resources - software, databases, teaching materials - maintained by members of the <em>CompbioZurich</em> community</li>
<li>job postings</li>
<li>information about teaching events and programs</li>
</ul>

<a href="/news/2019/2019-03-25-compbiozurich.org-now-on-github/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 20, 2019
      
    </span>
</aside>
<h3 class="title_news">Long fragments achieve lower base quality in Illumina paired-end sequencing</h3>
      
<h5 class="title_news">ZH Seminars Bioinformatics - Ge Tan</h5>
      
<!-- Use a hidden p tag to provide a preview -->
      <ul>
<li>Wednesday, 2019-03-20</li>
<li>12:15 in Y-55L-12, UZH Irchel campus</li>
</ul>
<p><em>Abstract</em> Illumina’s technology provides high quality reads of DNA fragments with error rates below 1/1000 per base. Sequencing runs typically generate millions of reads in which the vast majority of the reads has an average error rate below 1/1000. However, some paired-end sequencing data show the presence of a subpopulation of reads where the second read (R2) has lower average qualities. We show that the fragment length is a major driver of increased error rates in the R2 reads.
<a href="/news/2019/2019-03-20-zh-seminars-in-bioinformatics/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      January 29, 2019
      
    </span>
</aside>
<h3 class="title_news">Brupbacher Meeting 2019</h3>
      
<!-- Use a hidden p tag to provide a preview -->
      <p>The 14th Charles Rodolphe Brupbacher Symposium <strong>"Breakthroughs in Cancer Research and Therapy"</strong> will take place on January 29 - February 1, 2019. The event is co-organised with the <a href="https://www.uzh.ch">University of Zurich</a>.</p>

<a href="/news/2018-08-22-brupbacher-2019/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  
    <div class="blog-post-preview">
<aside class="blog-post-aside">
    <span>
      
      <span class="twemoji">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M6.75 0a.75.75 0 0 1 .75.75V3h9V.75a.75.75 0 0 1 1.5 0V3h2.75c.966 0 1.75.784 1.75 1.75v16a1.75 1.75 0 0 1-1.75 1.75H3.25a1.75 1.75 0 0 1-1.75-1.75v-16C1.5 3.784 2.284 3 3.25 3H6V.75A.75.75 0 0 1 6.75 0ZM21 9.5H3v11.25c0 .138.112.25.25.25h17.5a.25.25 0 0 0 .25-.25Zm-17.75-5a.25.25 0 0 0-.25.25V8h18V4.75a.25.25 0 0 0-.25-.25Z"/></svg>
      </span>
      March 12, 1989
      
    </span>
</aside>
<h3 class="title_news">Berners-Lee proposes the Web</h3>
      
<!-- Use a hidden p tag to provide a preview -->
      <p>Tim Berners-Lee makes a proposal for linking documents across servers.</p>
<a href="/news/1989-03-12-Berners-Lee-proposes-the-web/">
  <span class="twemoji">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13.22 19.03a.75.75 0 0 1 0-1.06L18.19 13H3.75a.75.75 0 0 1 0-1.5h14.44l-4.97-4.97a.749.749 0 0 1 .326-1.275.749.749 0 0 1 .734.215l6.25 6.25a.75.75 0 0 1 0 1.06l-6.25 6.25a.75.75 0 0 1-1.06 0Z"/></svg>
  </span>
  Continue reading
</a>
</div>
  


              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      &copy; 2022 Michael Baudis
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
        <div class="md-social">
  
    
    
    
    
      
      
    
    <a href="https://twitter.com/compbiozurich" target="_blank" rel="noopener" title="twitter.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"/></svg>
    </a>
  
    
    
    
    
      
      
    
    <a href="https://github.com/compbiozurich" target="_blank" rel="noopener" title="github.com" class="md-social__link">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link", "search.highlight", "search.share", "navigation.instant", "navigation.tracking", "navigation.sections", "toc.integrate"], "search": "../assets/javascripts/workers/search.a264c092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.6eac0284.min.js"></script>
      
    
  </body>
</html>